packages/local_auth/local_auth_android/lib/local_auth_android.dart

```diff
@@ -28,7 +28,7 @@ class LocalAuthAndroid extends LocalAuthPlatform {
     required Iterable<AuthMessages> authMessages,
     AuthenticationOptions options = const AuthenticationOptions(),
   }) async {
-    assert(localizedReason.isNotEmpty);
```

Comment:
@Luciferx86 I'm sorry I didn't catch this before, but I'm not sure this change is something we can safely make without changing the [platform interface](https://github.com/flutter/plugins/blob/ec2041f82584303d640fc7a3bc4ae31ebd76bc8e/packages/local_auth/local_auth_platform_interface/lib/local_auth_platform_interface.dart#L59), because this change assumed that the parameter is a nullable String (`String?`). 

It _may_ be a change that is possible, but we would need to see if it makes sense to make it a nullable parameter on the other platforms, as well.